cookbooks/continental_extension/continental_extension.prm

```diff
@@ -7,92 +7,144 @@
 #      Depth-dependent extension, two-stage breakup and cratonic underplating at rifted margins
 #   3. Naliboff, J., and Buiter, S.H. (2015), Earth Planet. Sci. Lett., v.421, p.58-67,
 #      "Rift Reactivation and migration during multiphase extension"
+#   4. Naliboff, J., Glerum, A., Sascha, S., Peron-Pinvidic, G., and Wrona, T. (2020), Geophys.
+#      Res. Lett., 47, e2019GL086611, "Development of 3‐D rift heterogeneity through fault
+#      network evolution"
+#   5. Sandiford, D., Brune, S., Glerum, A., Naliboff, J., and Whittaker, J.M. (2021), Geophys.
+#      Geochem. Geosys., 22, e2021GC009681, "Kinematics of footwall exhumation at oceanic
+#      detachment faults: Solid-block rotation and apparent unbending"
 
 ####  Global parameters
 set Dimension                              = 2
 set Start time                             = 0
 set End time                               = 5e6
 set Use years in output instead of seconds = true
-set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
 set Nonlinear solver tolerance             = 1e-4
-set Max nonlinear iterations               = 10
+set Max nonlinear iterations               = 100
 set CFL number                             = 0.5
+set Maximum time step                      = 20e3
 set Output directory                       = output-continental_extension
 set Timing output frequency                = 1
 set Pressure normalization                 = no
 
 #### Parameters describing the model
 
-# Model geometry (400x100 km, 2 km spacing)
+# Governing equations
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
+# Model geometry (200x100 km, 20 km spacing)
 subsection Geometry model
   set Model name = box
   subsection Box
-    set X repetitions = 200
-    set Y repetitions = 50
-    set X extent      = 400e3
+    set X repetitions = 10
+    set Y repetitions = 5
+    set X extent      = 200e3
     set Y extent      = 100e3
   end
 end
 
-# Mesh refinement specifications (no mesh refinement,
-# but the coarse mesh is already 200x50, see above)
+# Adaptively refine the mesh to 1.25 km spacing above y=50 km
+# and between x=40 and x=160 km. These values ensure areas
+# undergoing brittle deformation are in the high-resolution
+# region
 subsection Mesh refinement
-  set Initial adaptive refinement        = 0
-  set Initial global refinement          = 0
+  set Initial adaptive refinement        = 1
+  set Initial global refinement          = 2
```

Comment:
Yes, it's nicer to create the mesh via refinement than via the initial X/Y repetitions.